src/app/modules/main/wallet_section/view.nim

```diff
@@ -266,4 +268,34 @@ QtObject:
     self.delegate.resetRpcStats()
 
   proc canProfileProveOwnershipOfProvidedAddresses*(self: View, addresses: string): bool {.slot.} =
-    return self.delegate.canProfileProveOwnershipOfProvidedAddresses(addresses)
\ No newline at end of file
+    return self.delegate.canProfileProveOwnershipOfProvidedAddresses(addresses)
+
+  proc reloadAccountTokens*(self: View) {.slot.} =
+    self.delegate.reloadAccountTokens()
+
+  proc lastReloadTimestampChanged*(self: View) {.signal.}
+
+  proc setLastReloadTimestamp*(self: View, lastReloadTimestamp: int64) =
+    self.lastReloadTimestamp = lastReloadTimestamp
```

Comment:
You should put a guard here against assigning the same value, in order not to emit the signal if they are identical